string SAVE=save
string DELETE=delete
string GOTO=goto
string RENAME=rename
string command
string pose_name
string pose_name_new
